src/main/java/org/automation/pageObjects/GiftCardDashboardPage.java

```diff
@@ -15,4 +17,16 @@ public void clickOnConfigurationLink() {
 	public String getInfoMessage() {
 		return getText_custom(infoMessage);
 	}
+	
+	public void clickOnIssueAGiftCard() {
+		click(issueGiftCardBtn);
+	}
+
+	public boolean isIssueAGiftCardLinkPresent() {
+		return isElementPresent(issueGiftCardBtn,"Issue Gift Card Link");
```

Comment:
create Strings defined in class for error messages .

Reply:
It is Field name of Element.